ctapipe/tools/make_irf.py

```diff
@@ -0,0 +1,353 @@
+"""Tool to generate IRFs"""
+import operator
+
+import astropy.units as u
+import numpy as np
+from astropy.io import fits
+from astropy.table import vstack
+from pyirf.benchmarks import angular_resolution, energy_bias_resolution
+from pyirf.binning import create_histogram_table
+from pyirf.cuts import evaluate_binned_cut
+from pyirf.io import (
+    create_aeff2d_hdu,
+    create_background_2d_hdu,
+    create_energy_dispersion_hdu,
+    create_psf_table_hdu,
+    create_rad_max_hdu,
+)
+from pyirf.irf import (
+    background_2d,
+    effective_area_per_energy_and_fov,
+    energy_dispersion,
+    psf_table,
+)
+from pyirf.sensitivity import calculate_sensitivity, estimate_background
+
+from ..core import Provenance, Tool, traits
+from ..core.traits import Bool, Float, Integer, Unicode
+from ..irf import (
+    PYIRF_SPECTRA,
+    EventPreProcessor,
+    EventSelector,
+    GridOptimizer,
+    OptimisationResult,
+    OutputEnergyBinning,
+    SourceOffsetBinning,
+    Spectra,
+    ThetaCutsCalculator,
+)
+
+
+class IrfTool(Tool):
+    name = "ctapipe-make-irfs"
```

Comment:
Our terminology is a bit imprecises, but according to the DL3 data model document, the `IRF` is complete parametrization composed of *IRF components* (effective area, edisp, ...).

This tool computes thus a single IRF. Long story short, I'd call this `ctapipe-compute-irf` without the s.